utk/sass/_custom.scss

```diff
@@ -411,6 +411,84 @@ a.thumbnail:focus {
     font-weight: 500;
 }
 
+//
+//Flags
+//---------------------------------------------------
+.flag{
+   -moz-osx-font-smoothing: grayscale;
+    -webkit-font-smoothing: antialiased;
+    position: absolute;
+    top: 1.5em;
+    left: 1.5em;
+    float: left;
+    clear: both;
+    font-size:75%;     
+    padding: .2em 1.5em .3em;
+    border-radius: 0 50px 50px 0;
+    font-weight:500;
+    line-height: 1.25;
+    white-space: nowrap;
+    vertical-align: baseline;
+
+
+    &.flag-sale{
+      color:$color-clouds-lightest;
+      background: $color-red; 
+    }
+    &.flag-zodiac{
+      color:$color-clouds-lightest;
+      background: $color-zodiac; 
+    }
+    &.flag-mint{
+      color:$color-clouds-lightest;
+      background: $color-mint; 
+    }
+
+    &.flag-daisy{
+      color:$color-dove;
+      background: $color-daisy; 
+    }
+
+
+    // &.flag-circle{
```

Comment:
why comment this out?

Reply:
because it did not stay consistent with the style but i would like to keep it in incase we need to really call something out.